runhouse/servers/env_servlet.py

```diff
@@ -63,12 +61,19 @@ async def wrapper(*args, **kwargs):
 
 class EnvServlet:
     async def __init__(self, env_name: str, *args, **kwargs):
+        from runhouse.logger import logger
+
         self.env_name = env_name
 
+        logs_level = kwargs.get("logs_level", DEFAULT_LOG_LEVEL)
+        logger.setLevel(logs_level)
+        self.logger = logger
```

Comment:
why do we need to re-initialize a logger attribute here instead of using the global one?